scripts/apps/search/components/QueueError.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+/**
+ * @ngdoc React
+ * @module superdesk.apps.search
+ * @name QueueError
+ * @param {Object} item story
+ * @description Creates an icon for the published items that have been failed to enqueue.
+ */
+export class QueueError extends React.Component {
+    render() {
+        if (this.props.item.queue_state && this.props.item.queue_state === 'error') {
+            return (
+                <div className="icon-warning-sign queue-error" title={this.props.item.error_message} />
+            );
+        }
+
+        return null;
+    }
+}
+
+/*
+ * item: published item having error state
```

Comment:
You can delete the comment here and include at the top, in the `@param`. Then it will display on the docs page.